Sample Data/ASIM/CarbonBlackNotifications_CL_Schema.csv

```diff
@@ -0,0 +1,170 @@
+"threatInfo_threatCause_reputation_s",106,"System.String",string
+"threatInfo_threatCause_actor_s",107,"System.String",string
+"threatInfo_threatCause_actorName_s",108,"System.String",string
+"threatInfo_threatCause_actorProcessPPid_s",109,"System.String",string
+"threatInfo_threatCause_threatCategory_s",110,"System.String",string
+"threatInfo_threatCause_originSourceType_s",111,"System.String",string
+"threatInfo_threatCause_causeEventId_g",112,"System.String",string
+"threatInfo_threatCause_processGuid_s",113,"System.String",string
+"threatInfo_threatCause_parentGuid_s",114,"System.String",string
+"threatHunterInfo_incidentId_g",115,"System.String",string
+"threatHunterInfo_score_d",116,"System.Double",real
+"threatHunterInfo_summary_s",117,"System.String",string
+"threatHunterInfo_time_d",118,"System.Double",real
+"threatHunterInfo_indicators_s",119,"System.String",string
+"threatHunterInfo_watchLists_s",120,"System.String",string
+"threatHunterInfo_iocId_g",121,"System.String",string
+"threatHunterInfo_count_d",122,"System.Double",real
+"threatHunterInfo_dismissed_b",123,"System.SByte",bool
+"threatHunterInfo_documentGuid_s",124,"System.String",string
+"threatHunterInfo_firstActivityTime_d",125,"System.Double",real
+"threatHunterInfo_md5_g",126,"System.String",string
+"threatHunterInfo_policyId_d",127,"System.Double",real
+"threatHunterInfo_processGuid_s",128,"System.String",string
+"threatHunterInfo_processPath_s",129,"System.String",string
+"threatHunterInfo_reportName_s",130,"System.String",string
+"threatHunterInfo_reportId_s",131,"System.String",string
+"threatHunterInfo_reputation_s",132,"System.String",string
+"threatHunterInfo_responseAlarmId_g",133,"System.String",string
+"threatHunterInfo_responseSeverity_d",134,"System.Double",real
+"threatHunterInfo_runState_s",135,"System.String",string
+"threatHunterInfo_sha256_s",136,"System.String",string
+"threatHunterInfo_targetPriority_s",137,"System.String",string
+"threatHunterInfo_threatCause_reason_s",138,"System.String",string
+"threatHunterInfo_threatCause_actorProcessPPid_s",139,"System.String",string
+"threatHunterInfo_threatCause_parentGuid_s",140,"System.String",string
+"threatHunterInfo_threatCause_causeEventId_s",141,"System.String",string
+"threatHunterInfo_threatCause_reputation_s",142,"System.String",string
+"threatHunterInfo_threatCause_actor_s",143,"System.String",string
+"threatHunterInfo_threatCause_actorName_s",144,"System.String",string
+"threatHunterInfo_threatCause_threatCategory_s",145,"System.String",string
+"threatHunterInfo_threatCause_originSourceType_s",146,"System.String",string
+"threatHunterInfo_threatCause_processGuid_s",147,"System.String",string
+"threatHunterInfo_threatId_g",148,"System.String",string
+"threatHunterInfo_lastUpdatedTime_d",149,"System.Double",real
+"threatHunterInfo_orgId_d",150,"System.Double",real
+"url_s",151,"System.String",string
+"type_s",152,"System.String",string
+"eventDescription_s",153,"System.String",string
```

Comment:
Seems like we can extract other threat information from this field(threatInfo_indicators_s) in threat logs e.g. ThreatName. can you check?

Reply:
Not possible.

Reply:
@jayeshprajapaticrest @niralishah-crest 
why? can't we extract required info from threatInfo_indicators_s field which is in json format? may be using parse_json?

Reply:
The issue is not with the extraction but with indicators_s fields containing the keys as below:
mApplicationName, mSha256Hash, mIndicatorName which are more likely to give information related to threat indicators.